uiplib/Wallpaper.py

```diff
@@ -38,14 +40,14 @@ def set_wallpaper_windows(self, filename):  # pragma: no cover
 
     def set_wallpaper_osx(self, filename):  # pragma: no cover
         """Set a file as OSX Wallpaper."""
-        from appscript import app, mactypes  # use applescript modules
+        command = ("osascript -e 'tell application \"System Events\" "
```

Comment:
Could be done easily. Although, I don't think we've had arranged for changing wallpapers per desktop in the previous version. I'll incorporate that in the next PR. :p